tests/tests.js

```diff
@@ -3441,6 +3441,27 @@ exports.defineAutoTests = function () {
                 }
             });
         });
+        describe('resolveLocalFileSystemURL on cdvfile://', function () {
+            it("file.spec.147 should be able to resolve cdvfile applicationDirectory fs root", function(done) {
+                var cdvfileApplicationDirectoryFsRootName;
+                if (cordova.platformId === 'android') {
+                    cdvfileApplicationDirectoryFsRootName = 'assets';
+                } else if (cordova.platformId === 'ios') {
+                    cdvfileApplicationDirectoryFsRootName = 'bundle';
+                } else {
```

Comment:
I guess since "assets" is already defined and working with resolveLocalFileSystemURL we can't change it.

---
In reply to: [64836116](https://github.com/apache/cordova-plugin-file/pull/182#discussion_r64836116) [](ancestors = 64836116)